libs/category/src/drivers/magento/transformers/applied-filter-transformer.service.ts

```diff
@@ -16,9 +16,18 @@ export class DaffMagentoAppliedFiltersTransformService {
 		};
 
 		daffFilters.forEach(filter => {
-			magentoFilters[filter.name] = {
-				...magentoFilters[filter.name],
-				[this.transformActionEnum(filter.action)]: filter.value
+			if(filter.action === DaffCategoryFilterActionEnum.FromTo) {
+				const fromToValues = filter.value.split('-');
+				magentoFilters[filter.name] = {
+					...magentoFilters[filter.name],
+					[MagentoCategoryFilterActionEnum.From]: fromToValues[0],
+					[MagentoCategoryFilterActionEnum.To]: fromToValues[1]
+				}
```

Comment:
I think you should add a brief comment here explaining why `DaffCategoryFilterActionEnum.FromTo` needs such special treatment.

Reply:
done